lib/ReactViewModels/MouseCoords.js

```diff
@@ -75,9 +76,15 @@ export default class MouseCoords {
                 const maxHeightGeoid = maxHeight + (this.geoidModel ? this.geoidModel.maximumHeight : 0.0);
                 errorBar = Math.max(Math.abs(approximateHeight - minHeightGeoid), Math.abs(maxHeightGeoid - approximateHeight));
             }
+            Cartographic.clone(position, this.lastHeightSamplePosition);
+            const terrainProvider = globe.terrainProvider;
 
             this.cartographicToFields(intersection, errorBar);
-            this.debounceSampleAccurateHeight(globe, intersection);
+            if (!(terrainProvider instanceof EllipsoidTerrainProvider)) {
+                this.debounceSampleAccurateHeight.cancel();
```

Comment:
Is this cancel necessary?  I think calling the debounce function again will automatically reset the debounce timer.